apps/web/src/actions/prompts/run.ts

```diff
@@ -35,7 +35,6 @@ export const runPromptAction = authProcedure
 
       return {
         output: stream.value,
-        response: run.response,
```

Comment:
This fix this error
https://latitude-l5.sentry.io/issues/6008744834/events/827e54994bab428dbff8b7c5df8f8e40/?project=4507922531418112